File: lab/python/resultado_25_dividido_por5/vdc.py
def is0(n2):
    return n2 == 0


def retornar_divisao(n1, n2):
    if not is0(n2):
        return n1 / n2
    raise ZeroDivisionError("ERRO de divisão por 0!")


print(retornar_divisao(25, 5))  # Deve retornar 5
print(retornar_divisao(25, 0))

File: lab/javascript/resultado_25_dividido_por5/vdc.js
function isNot0(n2){
    return n2 != 0;
}


function retornar_divisao(n1, n2){
    if (isNot0(n2)){
        return n1 / n2
    }
    throw new Error("ERRO de divisão por 0!")

}

console.log(retornar_divisao(25, 5))  // Deve retornar 5
console.log(retornar_divisao(25, 0))
